cli/cmds_issues.go

```diff
@@ -0,0 +1,153 @@
+package cli
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/katbyte/ghp-pr-sync/lib/gh"
+	"github.com/spf13/cobra"
+	//nolint:misspell
+	c "github.com/gookit/color"
+)
+
+func CmdIssues(_ *cobra.Command, _ []string) error {
+
+	// For each repo get all issues and add to project only bugs
+	//Can't add all issues with current limit on number of issues on a project
+	f := GetFlags()
+	p := gh.NewProject(f.Owner, f.ProjectNumber, f.Token)
+
+	c.Printf("Looking up project details for <green>%s</>/<lightGreen>%d</>...\n", f.Org, f.ProjectNumber)
+	project, err := p.GetProjectDetails()
+	if err != nil {
+		c.Printf("\n\n <red>ERROR!!</> %s", err)
+		return nil
+	}
+	pid := project.Data.Organization.ProjectV2.Id
+	c.Printf("  ID: <magenta>%s</>\n", pid)
+
+	statuses := map[string]string{}
+	fields := map[string]string{}
+
+	for _, f := range project.Data.Organization.ProjectV2.Fields.Nodes {
+		fields[f.Name] = f.Id
+		c.Printf("    <lightBlue>%s</> <> <lightCyan>%s</>\n", f.Name, f.Id)
+
+		if f.Name == "Status" {
+			for _, s := range f.Options {
+				statuses[s.Name] = s.Id
+				c.Printf("      <blue>%s</> <> <cyan>%s</>\n", s.Name, s.Id)
+			}
+		}
+	}
+	fmt.Println()
+	//For issues, only 1 repo will be passed so below will only use f.Repos[0]
+	r := gh.NewRepo(f.Owner, f.Repos[0], f.Token)
```

Comment:
is there a reason to limit it to 1 unlike prs? should be straight forward to just loop here like with PRs